src/components/Navigation/components/Item/Item.tsx

```diff
@@ -262,35 +294,6 @@ export function Item({
       {secondaryNavigationMarkup}
     </li>
   );
-
```

Comment:
The internal click handler needed to move up so that it's declared before used on line 233 (now 265).